mapie/regression/regression.py

```diff
@@ -507,7 +511,9 @@ def fit(
             self.verbose
         )
         # Fit the prediction function
-        self.estimator_ = self.estimator_.fit(X, y, sample_weight)
+        self.estimator_ = self.estimator_.fit(
+            X, y, sample_weight, **fit_params
+            )
```

Comment:
Just a writing style suggestion:
```suggestion
        )
```